src/modal/modal.js

```diff
@@ -334,11 +334,14 @@ class Modal extends React.Component<ModalPropsT, ModalStateT> {
     if (!this.props.isOpen && !this.state.isVisible) {
       return null;
     }
-    const mountNode = this.getMountNode();
-    if (!mountNode) {
-      return null;
-    }
-    return ReactDOM.createPortal(this.renderModal(), mountNode);
+    return (
+      <Layer
+        mountNode={this.props.mountNode}
+        onMount={this.props.isOpen ? this.autoFocus : undefined}
```

Comment:
I can't find the class method `autoFocus`. was this change tested?

Reply:
It's tested with an e2e test. The method is on the line 199.

Reply:
ah sorry